### DISCUSSION TITLE
Install v3 via docker

### DISCUSSION THREAD

**Discussion by kimmoahola** [requarks/wiki#3833] [General]
Hello,

I see there is no docker images for the upcoming v3?

I is possible to put the v3 image available for us to install it?

I would like to install v3 (instead of installing v2 and then updating it to v3).

Thanks!

**Comment by Phil-hostwiki** [comment]
Well, Wiki.js 3.0 is still in development and not yet released.
According to the docs, 3.0 Beta is planned for Summer 2021 and final release in Fall 2021.

If you use Postgres as your database, you should be able to upgrade to 3.0 seamlessly when it is available.
